src/maps/hourly-parameter-fuel-flow.map.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { BaseMap } from '@us-epa-camd/easey-common/maps';
+import { HrlyParamFuelFlow } from '../entities/hrly-param-fuel-flow.entity';
+import { HrlyParamFuelFlow as HrlyParamFuelFlowWorkspace } from '../entities/workspace/hrly-param-fuel-flow.entity';
+import { HourlyParamFuelFlowDTO } from '../dto/hourly-param-fuel-flow.dto';
+
+@Injectable()
+export class HourlyParameterFuelFlowMap extends BaseMap<
+  HrlyParamFuelFlow | HrlyParamFuelFlowWorkspace,
+  HourlyParamFuelFlowDTO
+> {
+  public async one(
+    entity: HrlyParamFuelFlow | HrlyParamFuelFlowWorkspace,
+  ): Promise<HourlyParamFuelFlowDTO> {
+    const monitoringSystemId = entity.monitorSystem?.monitoringSystemId ?? null;
+
+    return {
+      id: entity.id,
+      userId: entity.userId,
+      monitoringLocationId: entity.monitoringLocationId,
+      parameterCode: entity.parameterCode,
+      parameterValueForFuel: entity.parameterValueForFuel,
+      formulaIdentifier: entity.formulaIdentifier,
```

Comment:
the formulaIdentifier should come from the monitorFormula table and then monitoringFormulaRecordId needs to be added which returns the actual primary key of the table. The entity column name can be changed to monitoringFormulaId (mon_form_id) for clarity or left the same. 
monitoringFormulaId: entity.formulaIdentifier